src/api/parser/package.json

```diff
@@ -31,8 +34,11 @@
     "node": ">=14.0.0"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "1.0.0",
     "babel-jest": "27.5.1",
     "env-cmd": "10.1.0",
-    "nodemon": "2.0.15"
+    "eslint": "7.32.0",
+    "nodemon": "2.0.15",
+    "supertest": "6.1.6"
```

Comment:
I think something happened when you were rebasing. We don't need supertest in the parser service

Reply:
I stand corrected. ESLint actually likes having supertest as a dep. We also use it for tests